isis/src/tgo/tsts/coloredMosaic/Makefile

```diff
@@ -0,0 +1,149 @@
+	$(LS) $(OUTPUT)/*bluEqui.cub > $(OUTPUT)/bluMosaic.lis;
+	$(MOSAIC) $(TSTSARGS) \
+  	fromlist=$(OUTPUT)/bluMosaic.lis \
+  	mosaic=$(OUTPUT)/bluCassisMosaic.cub \
+  	grange='USER' \
+  	MAXLAT = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MaximumLatitude grpname=Mapping`  \
+  	MINLAT = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MinimumLatitude grpname=Mapping`  \
+  	MAXLON = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MaximumLongitude grpname=Mapping` \
+  	MINLON = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MinimumLongitude grpname=Mapping` > /dev/null;
+
+	# Create Pan Mosaic
+	$(PROJECT) $(TSTARGS) from=$(OUTPUT)/$\$$\1.cub to=$(OUTPUT)/$\$$\1_panEqui.cub \
+  	map=$(OUTPUT)/equi.map \
+  	pixres=mpp resolution=200 -batchlist=$(OUTPUT)/panBasenames.lis > /dev/null;
+	$(LS) $(OUTPUT)/*panEqui.cub > $(OUTPUT)/panMosaic.lis;
+	$(MOSAIC) $(TSTSARGS) \
+  	fromlist=$(OUTPUT)/panMosaic.lis \
+   	mosaic=$(OUTPUT)/panCassisMosaic.cub \
+   	grange='USER' \
+   	MAXLAT = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MaximumLatitude grpname=Mapping`  \
+   	MINLAT = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MinimumLatitude grpname=Mapping`  \
+   	MAXLON = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MaximumLongitude grpname=Mapping` \
+   	MINLON = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MinimumLongitude grpname=Mapping` > /dev/null;
+
+	# Create Nir Mosaic
+	$(PROJECT) $(TSTARGS) from=$(OUTPUT)/$\$$\1.cub to=$(OUTPUT)/$\$$\1_nirEqui.cub \
+  	map=$(OUTPUT)/equi.map \
+  	pixres=mpp resolution=200 -batchlist=$(OUTPUT)/nirBasenames.lis > /dev/null;
+	$(LS) $(OUTPUT)/*nirEqui.cub > $(OUTPUT)/nirMosaic.lis;
+	$(MOSAIC) $(TSTSARGS) \
+  	fromlist=$(OUTPUT)/nirMosaic.lis \
+  	mosaic=$(OUTPUT)/nirCassisMosaic.cub \
+  	grange='USER' \
+  	MAXLAT = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MaximumLatitude grpname=Mapping`  \
+  	MINLAT = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MinimumLatitude grpname=Mapping`  \
+  	MAXLON = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MaximumLongitude grpname=Mapping` \
+  	MINLON = `$(GETKEY) from=$(OUTPUT)/equi.map keyword=MinimumLongitude grpname=Mapping` > /dev/null;
+
+	# Create full color mos
+	$(LS) $(OUTPUT)/*CassisMosaic.cub > $(OUTPUT)/mosaicList.txt;
+	$(CUBEIT) fromlist=$(OUTPUT)/mosaicList.txt to=$(OUTPUT)/coloredMosaic.cub > /dev/null;
+	$(EXPORT) $(TSTSARGS) from=$(OUTPUT)/coloredMosaic.cub to=$(OUTPUT)/coloredMosaic \
+		pdsversion=pds4 > /dev/null;
+
+	# Cleanup
+	$(RM) $(OUTPUT)/*B1*cub;
+	$(RM) $(OUTPUT)/*.txt;
+	$(RM) $(OUTPUT)/*.xml;
+	$(RM) $(OUTPUT)/*.img;
+	$(RM) $(OUTPUT)/*.lis;
+	$(RM) $(OUTPUT)/bluCassisMosaic.cub;
+	$(RM) $(OUTPUT)/nirCassisMosaic.cub;
+	$(RM) $(OUTPUT)/panCassisMosaic.cub;
+	$(RM) $(OUTPUT)/redCassisMosaic.cub;
+
+
+	$(RM) $(OUTPUT)/*CassisMosaic.xml;
+	$(RM) $(OUTPUT)/*CassisMosaic1.txt;
+	$(RM) $(OUTPUT)/*CassisMosaic2.txt;
+	$(RM) $(OUTPUT)/*CassisMosaic3.txt;
+	$(RM) $(OUTPUT)/coloredMosaic.img;
+	$(RM) $(OUTPUT)/coloredMosaic.xml;
```

Comment:
Don't remove the output IMG and XML files.  The XML file will need to be modified to remove some things and renamed to a TXT file.